README.md

```diff
@@ -76,19 +90,36 @@ Ideas for future work:
 [package:gql_pedantic:source]: ./gql_pedantic/README.md
 [package:gql_pedantic]: https://pub.dartlang.org/packages/gql_pedantic
 [package:gql_pedantic:version]: https://img.shields.io/pub/v/gql_pedantic.svg?style=flat-square 
+[package:multipack:source]: ./multipack/README.md
+[package:multipack]: https://pub.dartlang.org/packages/multipack
+[package:multipack:version]: https://img.shields.io/pub/v/multipack.svg?style=flat-square 
+[package:gql_example_cli:source]: ./gql_example_cli/README.md
+[package:gql_example_cli]: https://pub.dartlang.org/packages/gql_example_cli
+[package:gql_example_cli:version]: https://img.shields.io/pub/v/gql_example_cli.svg?style=flat-square 
+[package:gql_example_flutter:source]: ./gql_example_flutter/README.md
+[package:gql_example_flutter]: https://pub.dartlang.org/packages/gql_example_flutter
+[package:gql_example_flutter:version]: https://img.shields.io/pub/v/gql_example_flutter.svg?style=flat-square 
 
 
 ## Users
 
 | Project | Description |
 |---------|-------------|
 | [artemis][artemis] | Build dart types from GraphQL schemas and queries (using Introspection Query). | 
-| [graphql fork][graphql_flutter] | A stand-alone GraphQL client for Dart, bringing all the features from a modern GraphQL client to one easy to use package. |
-| [graphql_flutter fork][graphql_flutter] | A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [graphql][graphql_flutter] | A stand-alone GraphQL client for Dart, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [graphql_flutter][graphql_flutter] | A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package. |
+| [normalize][normalize] | Normalization and denormalization of GraphQL responses in Dart |
+| [gql_client][gql_client] | GraphQL Client for Dart |
+| [graphql-to-dart][graphql-to-dart]| generate dart classes and respective JsonSerializable transcoders |
+| [graphql_server][graphql_server]| Base package for implementing GraphQL servers. |
```

Comment:
@thosakwe, is this the only package currently directly using any of the packages in this repo?

Reply:
Technically, `angel_graphql` uses `gql`, BUT it's indirect - it only depends on `graphql_server` directly. So, I would say yes, `graphql_server` is the only one.

Reply:
Thanks for clarification!